input/intro-notes/StructureDefinition-au-core-allergyintolerance-notes.md

```diff
@@ -11,20 +11,14 @@
         <td>patient</td>
         <td><b>SHALL</b></td>
         <td><code>reference</code></td>
-        <td>The requester <b>SHALL</b> provide at least an id value and <b>MAY</b> provide both the Type and id values. The responder <b>SHALL</b> support both.</td>
+        <td>The requester <b>SHALL</b> provide at least an id value and <b>MAY</b> provide both the Type and id values. The responder <b>SHALL</b> support both.<br/><br/>The requester <b>SHALL</b> provide both the system and code values. The responder <b>SHALL</b> support both.<br/><br/> The requester <b>SHOULD</b> support chained search patient.identifier using IHI, Medicare Number, and DVA Number identifiers as defined in the AU Core Patient profile. The responder <b>SHOULD</b> support chained search patient.identifier using IHI, Medicare Number, and DVA Number identifiers as defined in the AU Core Patient profile.</td>
```

Comment:
**This comment applies to ALL profiles included in this PR:**
"The requester SHALL provide both the system and code values. The responder SHALL support both." which comes from the chained search, now reads as though it belongs to patient.  Suggest linking it to the chained search detail - either by placing under chained search paragraph or in same paragraph. 
This text is not in any of the Requester or Responder CapabilityStatements for profiles listed in this PR.